pom.xml

```diff
@@ -75,6 +75,11 @@
           <version>1.1.1</version>
           <scope>compile</scope>
         </dependency>
+        <dependency>
+		    <groupId>org.owasp.encoder</groupId>
```

Comment:
What this new dependency does ?

Reply:
In several classes the following issue has been detected: